src/common/search/SearchService.js

```diff
@@ -19,7 +19,7 @@
       searchlayer_ = new ol.layer.Vector({
         metadata: {
           title: $translate.instant('search_results'),
-          internalLayer: true
+          searchLayer: true
```

Comment:
I could not see anywhere else besides loading the features that internalLayer was used - is there any danger to modifying this?